pkg/server/http/server.go

```diff
@@ -13,6 +13,7 @@ import (
 
 var log = logging.Logger("lassie/httpserver")
 
+// A Lassie server for fetching data from the network via HTTP
```

Comment:
your godoc comments need to start with the thing you're documenting

```suggestion
// HttpServer is a Lassie server for fetching data from the network via HTTP
```